extensions/sql-database-projects/src/test/baselines/openSqlProjectBaseline.xml

```diff
@@ -94,6 +94,14 @@
       <DatabaseVariableLiteralValue>master</DatabaseVariableLiteralValue>
     </ArtifactReference>
   </ItemGroup>
+  <ItemGroup>
+    <PreDeploy Include="Script.PreDeployment1.sql"/>
+    <None Include="Script.PreDeployment2.sql"/>
+  </ItemGroup>
+  <ItemGroup>
+    <PostDeploy Include="Script.PostDeployment1.sql"/>
+    <None Include="Tables\Script.PostDeployment1.sql"/>
+  </ItemGroup>
```

Comment:
Does DacFx/SqlToolsService know to look for scripts with these specific names at deployment time, or is that part not yet hooked up?

Reply:
Yes, that part already exists in DacFx/SqlToolsService. 
I verified that Project > Publish works for pre/post deployment scripts too.